src/operator/tensor/matrix_op.cu

```diff
@@ -22,11 +22,111 @@
  * \file matrix_op.cu
  * \brief GPU Implementation of matrix operations
  */
+#include <cub/cub.cuh>
 #include "./matrix_op-inl.h"
 #include "./elemwise_unary_op.h"
 
+
 namespace mxnet {
 namespace op {
+
+
+// compute the number of elements of every row
+struct SliceMarkCsrIndPtr {
```

Comment:
Please add brief documentation for the parameters of the kernel

Reply:
Added